### PR TITLE
Add password strength indicator in register form

### LINKED ISSUE #4937: Add password strength indicator
https://github.com/dropbox/zxcvbn

### PR DESCRIPTION
**Pull request by billyvg**
(This is a WIP)

1) zxcvbn is a really big lib to load clientside, this async loads it in `organization-login` template (do we need to load it in `login` template, are there others?). Is this ok? cc @macqueen @MaxBittker 

@ckj 
2) Do we want a) a bar, b) "strength" label, c) both?
3) The library also provides password warnings as well as suggestions.
I think to keep it lightweight we just have the strength text + warnings. I think we'll want warnings because you could have a long password with repeated substrings and it'll have a "Weak" strength score.

Closes #4937

### CODE REVIEW

**Comment by getsentry-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">You should update CHANGES due to the size of this PR</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

File: src/sentry/templates/sentry/organization-login.html
Comment (benvinegar) [inline src/sentry/templates/sentry/organization-login.html]: I wouldn't bother with this – we do not support below IE11.

**Comment by MaxBittker** [comment]
I think it would be nice to avoid vendoring this. it's good to have it async, but what about having this be it's own (tiny+simple) entry point +bundle in webpack? that would also mean less javascript inside the template. just worth considering imo

File: src/sentry/templates/sentry/organization-login.html
Comment (benvinegar) [inline src/sentry/templates/sentry/organization-login.html]: Not used to seeing IIFEs with `.call` ... is there a benefit to this over just doing `})();`?

File: src/sentry/static/sentry/js/zxcvbn.js
Comment (macqueen) [inline src/sentry/static/sentry/js/zxcvbn.js]: can we put this in the `vendor` directory? (same level as `js`)

**Review by macqueen — COMMENTED** [review]
the other place we'll want this is here: https://sentry.io/signup/ which is in the getsentry repo

File: src/sentry/templates/sentry/organization-login.html
Comment (billyvg) [inline src/sentry/templates/sentry/organization-login.html]: Yeah there's no benefit, was just C&P from zxcvbn's docs.

**Comment by billyvg** [comment]
zxcvbn is ~400KB min/gzipped, here's how this currently looks like a simulated load latency of ~5s:

pw-str

**Comment by ckj** [comment]
@billyvg Looks great! I'm happy with having both the bar and label. Nitpicks: I'd probably soften the edges of the bar with `border-radius: 2px` as well as add a little margin between the Password Strength line and the bar.

Another great feature! 💥

**Comment by dcramer** [comment]
<3 it

**Comment by benvinegar** [comment]
Are we merging this?

**Comment by billyvg** [comment]
I think we should?

File: webpack.config.js
Comment (MaxBittker) [inline webpack.config.js]: I think that this is kinda a breaking point for our webpack config to be become multiple files. 

maybe keep webpack.config.js where it is and add a webpack/ dir next to it with stuff like this we can import?

**Review by MaxBittker — APPROVED** [review]
the behavior is really nice, seems safe to merge and is a good improvement- but it's slightly ugly imo for something people see so early in the signup flow. (not blocking on this since chris approved and we can easily circle back with css)

the webpack thing is maybe a bigger discussion, the file has been growing fast enough that it's becoming hard to find things inside it. could be a follow up to refactor it

File: webpack.config.js
Comment (billyvg) [inline webpack.config.js]: Definitely agree, the storybook config also creates another entry point, so once everything gets merged in I'll work on refactoring the conf.

Comment (MaxBittker) [inline webpack.config.js]: sweet, let me know if you want to pair on that

**Pull request merged**
